lib/connect-session-sequelize.js

```diff
@@ -25,7 +29,20 @@ module.exports = function SequelizeSessionInit(Store) {
 		}
 		Store.call(this, options);
 
+		var key;
+		this.options = {};
+		for (key in defaultOptions) {
```

Comment:
I don't really understand what you're doing  with these `defaultOptions`? Couldn't you use a `extend`, e.g.:
```javascript
util._extend({foo: 'bar'},{bar:'baz',foo:'x'})
```